### PR TITLE
Update userguide for Servlet 5

### PR DESCRIPTION
**Pull request by arjantijms**
I'm not sure if some of these changes need to be done locally in the servlet folder, or more globally in the Templates folder. If this is not correct, please advise.

cc @kwsutter 

Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: user_guides/servlet/src/main/jbake/content/config.adoc [hunk 1]
```diff
@@ -58,7 +58,7 @@ with the necessary configuration steps for your implementation and place
 the file under the `<TS_HOME>/bin/xml/impl/<your_impl>` directory.
 
 For more information, you may wish to view `<TS_HOME>/bin/xml/impl/glassfish/config.vi.xml`,
-the configuration file for Eclipse EE4J Jakarta EE 8 Compatible Implementation, Eclipse GlassFish.
+the configuration file for Eclipse EE4J Jakarta EE 9 Compatible Implementation, Eclipse GlassFish.
```
Comment (scottmarlow) [inline user_guides/servlet/src/main/jbake/content/config.adoc:61]: Should we remove ```Eclipse EE4J```?  

FYI, I see around 65 references to ```Eclipse EE4J``` in the jakartaee-tck source and 3 references to just ```EE4J```.

Comment (scottmarlow) [reply user_guides/servlet/src/main/jbake/content/config.adoc:61]: @arjantijms as per discussion on [Required Fixes on the naming of Jakarta EE across its mediums] It is NOT Eclipse Jakarta EE =/ It is Jakarta EE](https://www.eclipse.org/lists/jakarta.ee-community/msg02411.html), we should change "Eclipse EE4J Jakarta EE" to "Jakarta EE".

Could you please update your pull request to use "Jakarta EE 9".  Thank you!

Scott

Comment (arjantijms) [reply user_guides/servlet/src/main/jbake/content/config.adoc]: Thanks for the extra pair of eyes Scott, done!

**Comment by arjantijms** [comment]
@edbratt 
>I believe CentOS 7 is the only OS tested

The version that the CI reports when I run the standalone TCK on the CI that has been given to our project is "Alpine Linux v3.12". See: https://ci.eclipse.org/servlet/job/2_servlet-run-tck-against-staged-build/lastSuccessfulBuild/artifact/servlet-tck/bin/summary.txt

**Comment by arjantijms** [comment]
@edbratt I updated the issues you mentioned and found a lot more instances where javax.* and old terms like EJB where used, and update these.

Can you take a look again? Thx!

File: internal/docs/servlet/README.txt [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 
-1) Java SE 7
-2) Servlet RI Version 4.0
-3) Servlet TCK Version 4.0
+1) Java SE 8
+2) Servlet Compatible Implementation Version 5.0
+3) Servlet TCK Version 5.0
 
-1. Download and install Java SE 7 software.
+1. Download and install Java SE 8 or Java 11 software.
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply internal/docs/servlet/README.txt:15]: I think I got this from another location in the Servlet files where this was stated, but I'll remove it to be sure. Thx!

File: install/servlet/docs/index.html [hunk 3]
```diff
@@ -38,30 +38,30 @@
...
+	  <p>The <a href="ReleaseNotes-servlet.html">Jakarta</a><a href="javatest/javatest.pdf"><em><small><sup></sup></small></em></a><a href="ReleaseNotes-servlet.html"> Servlet TCK 5.0 Release Notes</a> provide the information that you need to install, set up, and run the Jakarta Servlet TCK.</p>
 	</li>
 	<li>
-	  <p>The <em>Jakarta Servlet Technology Compatibility Kit 4.0 User's Guide</em> provides the information you need to install, configure, and run the Jakarta Servlet TCK. The User's Guide also contains the rules you need to pass for certification.</p>
+	  <p>The <em>Jakarta Servlet Technology Compatibility Kit 5.0 User's Guide</em> provides the information you need to install, configure, and run the Jakarta Servlet TCK. The User's Guide also contains the rules you need to pass for certification.</p>
 	</li>
 	<li>
-	  <p>The <a href="assertions/api-assertions.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the Jakarta Servlet TCK.</p></li>
+	  <p>The <a href="assertions/ServletSpecAssertions.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the Jakarta Servlet TCK.</p></li>
```
Comment (alwin-joseph) [inline install/servlet/docs/index.html:56]: Did you mean ServletJavadocAssertions.html here

File: install/servlet/docs/index.html [hunk 4]
```diff
@@ -38,30 +38,30 @@
...
 	<li>
-	  <p>The <a href="assertions/spec-assertions.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Jakarta Servlet TCK.</p></li>
+	  <p>The <a href="assertions/ServletJavadocAssertions.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Jakarta Servlet TCK.</p></li>
```
Comment (alwin-joseph) [inline install/servlet/docs/index.html:58]: Did you mean ServletSpecAssertions.html here

**Comment by scottmarlow** [comment]
Imo, this is definitely needed, I just don't want to merge it until we hear back from @arjantijms

**Comment by edbratt** [comment]
@arjantijms The template mechanism was intended just to be a one-release stop-gap. But no effort was ever allocated to improving it since the plan was to distribute (refactor) production of these TCKs out to the API projects -- that hasn't happened either. I guess this is, once again proving: there is nothing more permanent than a temporary fix. Sorry for the frustration.

File: install/servlet/docs/index.html (hunk 3 above)
Comment (arjantijms) [reply install/servlet/docs/index.html:56]: Good catch, I'll correct them!

**Comment by arjantijms** [comment]
> Imo, this is definitely needed, I just don't want to merge it until we hear back from @arjantijms and additional review comments are resolved.

I addressed the review comments. I think I can be merged now.

**Comment by alwin-joseph** [comment]
> > Imo, this is definitely needed, I just don't want to merge it until we hear back from @arjantijms and additional review comments are resolved.
> 
> I addressed the review comments. I think I can be merged now.

Thanks Arjan

**Pull request merged**